### ISSUE TITLE
[joomla.org] 

### ISSUE THREAD

**Issue by FxBullMarket** [joomla/joomla-websites#1800]
Hello. They can 't solve my problem on the forum . What should I do now?????

**Comment by drmenzelit** [comment]
What is your problem?

**Comment by FxBullMarket** [comment]
> В чем ваша проблема?

Hello, I need help, after switching from 3.10.x to joomla 4 (version 4.2.1) for some reason, registered users can log in to the site, but new ones cannot register. I removed the old plugins and modules because there was no version 4. I have temporarily enabled the standard joomla component, but it gives a short error message "Registration failed" (Registration failed:). And after the transition there are many problems, but now it is more important.
The topic is open on the official Joomla forum, but no one can find a solution. We checked the site folders and the database on the hosting, everything is fine there
